### PR TITLE
WIP: Added nancy dependency scanner

### PR DESCRIPTION
**Pull request by Nerzal**
Tryin to add nancy dependency scanner.

https://github.com/sonatype-nexus-community/nancy

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,3 +37,11 @@ jobs:
     - stage: goreportcard
       script: curl --fail --request POST "https://goreportcard.com/checks" --data "repo=github.com/Nerzal/gocloak"
       if: branch = master AND type = push
+    - stage: nancy
```
Comment (SVilgelm) [inline .travis.yml]: I think you can put it into test stage

File: .travis.yml [hunk 2]
```diff
@@ -37,3 +37,11 @@ jobs:
...
+      before_script:
```
Comment (SVilgelm) [inline .travis.yml]: this should be `install` section

File: .travis.yml [hunk 3]
```diff
@@ -37,3 +37,11 @@ jobs:
...
+      - cd /tmp && mkdir tools && cd -
+      - sudo curl -s -L "https://github.com/sonatype-nexus-community/nancy/releases/download/0.0.5/nancy-linux.amd64-0.0.5" -o "/tmp/tools/nancy"
+      - sudo chmod +x /tmp/tools/nancy
+      - export PATH=$PATH:/tmp/tools/
```
Comment (SVilgelm) [inline .travis.yml]: you really don't need to export, you can directly call the nancy: /tmp/tools/nancy

File: .travis.yml [hunk 4]
```diff
@@ -37,3 +37,11 @@ jobs:
     - stage: goreportcard
       script: curl --fail --request POST "https://goreportcard.com/checks" --data "repo=github.com/Nerzal/gocloak"
       if: branch = master AND type = push
+    - stage: nancy
+      before_script:
+      - cd /tmp && mkdir tools && cd -
+      - sudo curl -s -L "https://github.com/sonatype-nexus-community/nancy/releases/download/0.0.5/nancy-linux.amd64-0.0.5" -o "/tmp/tools/nancy"
```
Comment (SVilgelm) [inline .travis.yml]: the lates version is v0.0.39

**Comment by SVilgelm** [comment]
if you are interesting, I added Nancy here: https://github.com/SVilgelm/oas3-server/blob/master/.github/workflows/go.yml#L48

Bu I use GitHub Actions for that project, not Travis CI

File: .travis.yml (hunk 4 above)
Comment (SVilgelm) [inline .travis.yml]: Also I think you don't really need a sudo here and later

**Comment by Nerzal** [comment]
> if you are interesting, I added Nancy here: https://github.com/SVilgelm/oas3-server/blob/master/.github/workflows/go.yml#L48
> 
> Bu I use GitHub Actions for that project, not Travis CI

Okay, i'll have a look into it, thanks!

**Comment by SVilgelm** [comment]
So, Nancy is here, lets close this PR

**Pull request closed without merging**